tests/test_conn.py

```diff
@@ -204,3 +204,12 @@ def invoke_osserror(*a, **kw):
         with self.assertRaises(ConnectionError):
             yield from conn.send(request)
         self.assertEqual(conn.connected(), False)
+
+    @run_until_complete
+    def test_close_disconnects_connection(self):
+        asyncio.set_event_loop(self.loop)
```

Comment:
Ok sorry about that I was just expanding test `test_global_loop_for_create_conn` and this remained there.